nav2_msgs/srv/GetMap3D.srv

```diff
@@ -0,0 +1,5 @@
+# Get the map as a sensor_msgs/PointCloud2
+---
+# The current map hosted by this map service.
+sensor_msgs/PointCloud2 map
```

Comment:
As in the comment above, if we do indeed need the origin, we should have a new message for a 3D map containing any pertenent information in addition to the PC2 that it needs (e.g. origin,  if voxelized, if binary, etc)

Reply:
Apply this comment to all 3 of these message definitions